utilities/private/ft_struct2table.m

```diff
@@ -1,13 +1,13 @@
-function table = struct2table(s)
+function table = ft_struct2table(s)
```

Comment:
Please rename this back; I prefer private functions not to be prefixed with ft\_

It makes the high-level code more messy, and they should not interfere with users' functions anyway.

Reply:
It needs to be renamed to something, because it is masking Matlab's own `struct2table`, which is needed for the table conversion.

How about `struct2tablestrs`?

Reply:
hmm, I did not notice that name clash before. But on https://nl.mathworks.com/help/matlab/ref/struct2table.html I see that it already exists since 2013b...

Oh, red flag: I realize that I am using them both. The one in utilities/private is only used in ft_test_xxx, all other uses of struct2table is using the Mathworks version.

Looking at my own struct2table I also find it a rather limited function, it only serves for pretty-printing. I suspect that at the time of implementation (which may predate 2017) I was not willing to use the MATLAB table object yet. Now I think it actually makes more sense to use an actual table object, since those display nicely.

Could you check whether we could get rid of my struct2table and simply use a proper table?

Reply:
Uh oh.

Could you check whether we could get rid of my struct2table and simply use a proper table?

We could do this, except for one thing: when you `disp(t)` a table, it includes ugly quotes around all the cellstrs:

```

>> rslt = ft_test('report', 'fieldtripversion', 'cef3396');
>> disp(rslt(1:10,:))
    matlabversion    fieldtripversion     branch       arch        hostname        user      passed    runtime        functionname     
    _____________    ________________    ________    _________    ___________    ________    ______    _______    _____________________
       '2016b'          'cef3396'        'master'    'glnxa64'    'dccn-c046'    'roboos'    true         2       'test_csp'           
       '2016b'          'cef3396'        'master'    'glnxa64'    'dccn-c041'    'roboos'    true         1       'test_bug840'        
       '2016b'          'cef3396'        'master'    'glnxa64'    'dccn-c036'    'roboos'    true         6       'test_bug1149'       
       '2016b'          'cef3396'        'master'    'glnxa64'    'dccn-c037'    'roboos'    true         4       'test_bug1315'       
       '2016b'          'cef3396'        'master'    'glnxa64'    'dccn-c036'    'roboos'    true         7       'test_bug1242'       
       '2016b'          'cef3396'        'master'    'glnxa64'    'dccn-c041'    'roboos'    true         1       'test_bug1881'       
       '2016b'          'cef3396'        'master'    'glnxa64'    'dccn-c047'    'roboos'    true         4       'test_ft_plot_lay'   
       '2016b'          'cef3396'        'master'    'glnxa64'    'dccn-c041'    'roboos'    true         1       'test_bug1910'       
```

But that's not a blocker. We could just replace your struct2table with a custom pretty-printing function that accepts Matlab `table` objects as input and produces the same output. That would be a good way to do it - table objects are really nice, so it'd be good to settle on passing them around instead of custom display structures.

Reply:
I have reverted the renames of the other uses of `struct2table` so they now keep calling Matlab's `struct2table`.